src/components/Flyout/Flyout.css

```diff
@@ -0,0 +1,39 @@
+.flyout {
+  position: fixed;
+  bottom: 20px;
+  left: 50%;
+  transform: translateX(-50%);
+  background: #333;
+  color: white;
+  padding: 12px 20px;
+  border-radius: 8px;
+  display: flex;
+  align-items: center;
+  gap: 16px;
+  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.2);
+  z-index: 1000;
+  font-size: 14px;
+}
+
+.flyout-buttons {
+  display: flex;
+  gap: 8px;
+}
+
+.flyout button {
```

Comment:
i would recommend you to use class names for styles, avoiding tags styling, for the whole project. i saw tags styling a lot in your project

e.g.:
`.flyout .flyout-button` instead of `.flyout button`